tstest/log.go

```diff
@@ -145,3 +145,50 @@ func (ml *MemLogger) String() string {
 	defer ml.Unlock()
 	return ml.Buffer.String()
 }
+
+// AsyncTestLogger returns a logger.Logf that logs to t.Logf until the test
```

Comment:
Is "Async" the right word here?

It's still synchronous while the test is running?

WhileTestRunningLogger? Verbose but clear.